testdata/get-vcs-relative-parent.txt

```diff
@@ -0,0 +1,54 @@
+# This is thame as get.txt except that we use the -vcs flag
```

Comment:
s/thame/the same/

(Yeah, I know it's my typo originally - perhaps fix the original one too?)

Also, it also differs because it uses a relative GOHACK dir (the point of this test) so perhaps worth mentioning that.

Reply:
Fixed in all locations, and comment updated.